addons/stock_dropshipping/models/stock_lot_customer.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class StockLotReport(models.Model):
+    _inherit = 'stock.lot.report'
+
+    def _join_on_picking_type_and_partner(self):
+        return """
+            JOIN stock_picking_type AS type
+            ON picking.picking_type_id = type.id and (type.code = 'outgoing' or type.code = 'dropship')
+            JOIN sale_order as so ON so.id = picking.sale_id
```

Comment:
I would left join here
Otherwise, if you simply deliver a lot to a customer through the inventory app, we will not see it
Can you add a test for that case ? Since the runbot is incorrectly green

(Side note: I guess it's ok for now to base the customer on SO, since you can't really do a dropship transfer through the inventory app)